app/controllers/timelogs_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TimelogsController < ApplicationController
+  before_action :load_user
+
+  def index
+    @timelog = Timelog.where(user_id: current_user.id)
+  end
+
+  def create
+    @timelog = @user.timelogs.create(timelog_params)
```

Comment:
`create` methods already triger sql to save the record in the db.
What you do here:
1. `. ...create` - triggers DB
2. `.save` - triggers DB.
It is better to `build object` and after that save.